### DISCUSSION TITLE
Is Campus Expert Programme Aug 2023 Ended already ?

### DISCUSSION THREAD

**Discussion by SALMANKHANPM** [community/community#62723] [GitHub Education]
### Hi everyone,

The last time I saw the Campus Expert page, it was like we will open in August. But now says Feb 2024. Does the Programme Application is already completed filling out?

**Comment by void-main1812** [comment]
I noticed the same thing

**Reply by ShadrackOdielo** [reply]
it is fixed now

**Comment by ankitpartap** [comment]
Yesturday it was showing "GitHub Campus Experts. Applications to the program will open in August 2023, and you'll have a full month to submit" 
Now it is showing this "GitHub Campus Experts. Applications to the program will open in February 2024, and you'll have a full month to submit"

**Comment by 8uraj** [comment]
i see same problem in my side also @githubeducation

**Comment by SALMANKHANPM** [comment]
> ### Hi everyone,
> The last time I saw the Campus Expert page, it was like we will open in August. But now says Feb 2024. Does the Programme Application is already completed filling out?
> 
> ### A suggestion...(optional)
> _No response_

Is this a glitch ?

**Reply by ShadrackOdielo** [reply]
A similar thing happened last time too

**Reply by ShadrackOdielo** [reply]
it is fixed now

**Reply by TammyAlok2** [reply]
It is showing it will end in 1 sep 2023 at 23:00 but it closed before the time
